main.cpp

```diff
@@ -195,6 +199,38 @@ void mergeImages()
   }
 }
 
+void FlipImage()
+{
+  cout << "Please choose\n"
+          "1->To flip the image horizontally\n"
+          "2->To flip the image vertically\n"
+          "your choice is: ";
```

Comment:
I think we can change the way we show the options to match what we've been using throughout the project.

```suggestion
  cout << "Flipping directions:\n"
       << "1. Flip horizontally\n"
       << "2. Flip vertically\n"
       << "Choose a flipping direction: ";
```